releasetasks/templates/mobile/candidates_fennec.yml.tmpl

```diff
@@ -15,7 +15,11 @@
              - docker-worker:cache:level-3-*
              - docker-worker:cache:tooltool-cache
              - scheduler:extend-task-graph:*
+             {% if branch in ('mozilla-release', 'mozilla-beta') %}
```

Comment:
@JohanLorenzo 

Maybe we should refine this even more since we're here:
if beta => use project-releng/candidates-fennec-beta
elif release => use project-releng/candidates-fennec-release
else => project-releng/candidates-fennec-dev

The release role encapsulates the beta role but security wise, it'd be nice to have that separation.

Reply:
That'd be nice, indeed. However there are things in common between beta and release. For instancce, beta is signed with the same key as release. Thus, I don't know what's the value of making the split.